vector/src/main/java/im/vector/app/features/home/room/detail/timeline/render/ProcessBodyOfReplyToEventUseCase.kt

```diff
@@ -100,10 +103,23 @@ class ProcessBodyOfReplyToEventUseCase @Inject constructor(
                         repliedToEvent.isPollEnd() -> stringProvider.getString(R.string.message_reply_to_sender_ended_poll)
                         else -> ""
                     }
+                    val repliedText = when {
```

Comment:
Maybe to have something easier to read, we should avoid nesting the when and handle `repliedToEvent.isPollStart()` and `repliedToEvent.isPollEnd()` separately.

Reply:
Refactored in [5820185](https://github.com/vector-im/element-android/pull/7969/commits/58201851ad196978f00df96bf1b76257a242c256)